server/models.py

```
@@ -0,0 +1,193 @@
+    patient_id = db.Column(db.String(50), db.ForeignKey('patients.id'))
+
+    clinic = db.relationship('Clinic', back_populates='bookings')
+    service = db.relationship('Service', back_populates='bookings')
+    patient = db.relationship('Patient', back_populates='bookings')
+
+    def __repr__(self):
+        return f'<Booking for {self.patient.name} at {self.clinic.name}>'
+
+
+
+class User(db.Model):
```

Flake8 (7.2.0)

[error] too many blank lines (3)

(E303)